packages/fxa-settings/src/components/UnitRow/index.tsx

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ import React, { useCallback } from 'react';
+ import { useBooleanState } from '@fxa-components/lib/hooks';
+ import Modal from '../Modal';
+
+ export const UnitRow = () => {
+  const [modalRevealed, revealModal, hideModal] = useBooleanState();
+
+  const onConfirmClick = useCallback(() => {
+    console.log('confirmed - do the thing');
```

Comment:
Hey!

Reply:
Do you want to keep this console log in place?

Reply:
When we actually have this hooked up, we'll have a function execute here, but right now we're not actually doing anything... I could take it out, but if someone pokes at this in the meantime, they wouldn't be able to tell the difference between dismissing and confirming.